packages/core/contracts/financial-templates/common/FundingRateApplier.sol

```diff
@@ -95,7 +128,7 @@ abstract contract FundingRateApplier is Testable, Lockable {
         view
         returns (FixedPoint.Unsigned memory tokenDebt)
     {
-        return rawTokenDebt.mul(cumulativeFundingRateMultiplier);
+        return rawTokenDebt.mul(getCumulativeFundingRateMultiplier());
```

Comment:
Does this mean that this method is now expected to be called without pre-updating the funding rate multiplier? Not necessarily a problem, but IIUC, where we use this, we would basically be running the _calculateEffectiveFundingRate function twice within the same transaction, right? Since we update the funding rate multiplier in a modifier that's called before the main logic begins?

Reply:
`getCumulativeFundingRateMultiplier()` is a helper method so that sponsors can more easily track the value of their debt in real time. The alternative is to call `applyFundingRate()` followed by `cumulativeFundingRateMultiplier()`. 

Moreover, yes, where wherever `_getFundingRateAppliedTokenDebt()` is used the computation will be done twice, but the state transition that stores a new value in `cumulativeFundingRateMultiplier()` is only done once. Look how `_applyEffectiveFundingRate()` differs from `getCumulativeFundingRateMultiplier()`.

This is my design at least, let me know if you think the implementation is diff.

Reply:
Thinking more about this. The main benefit of this feature is that it enhances funding-multiplier liveness: sponsors can get their up to date adjusted token debt value without having to send a transaction. However this would be a bit inconsistent with the fee-multiplier liveness, which we have no plans to offer the same view-only method that returns live data. I'm curious what the rest of the team thinks. 

@daywiss @chrismaree

Reply:
I dont think i really understand!

Reply:
Multiplier is supposed to reflect latest funding rate (% per second) which affects the value of sponsor debt. 

Multiplier is updated on every contract call. However, in between calls the multiplier itself hasn't changed but it's "intrinsic value" has changed because seconds continue to tick. 

The question is, is this "live" multiplier useful for sponsors, proposers, etc?